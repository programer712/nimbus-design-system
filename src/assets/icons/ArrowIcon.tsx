import { SvgIcon, SvgIconProps } from '@material-ui/core';
import React from 'react';

export function ArrowIcon(props?: SvgIconProps) {
  return (
    <SvgIcon {...props}>
      <path d="M12 20L13.41 18.59L7.83 13L20 13L20 11L7.83 11L13.42 5.42L12 4L4 12L12 20Z" />
    </SvgIcon>
  );
}
