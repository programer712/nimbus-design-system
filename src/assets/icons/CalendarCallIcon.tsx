import { SvgIcon, SvgIconProps } from '@material-ui/core';
import React from 'react';

export function CalendarCallIcon(props?: SvgIconProps) {
  return (
    <SvgIcon {...props}>
      <rect width="24" height="24" viewBox="0 0 24 24" fill="none" />
      <path
        fillRule="evenodd"
        clipRule="evenodd"
        d="M15.5556 4V5.2973H8.44444V4H6.66667V5.2973H5.77778C4.79111 5.2973 4.00889 6.07568 4.00889 7.02703L4 18.2703C4 19.2216 4.79111 20 5.77778 20H18.2222C19.2 20 20 19.2216 20 18.2703V7.02703C20 6.07568 19.2 5.2973 18.2222 5.2973H17.3333V4H15.5556ZM16.4443 13.0813H12.8888V16.5408H16.4443V13.0813ZM5.77849 18.2702H18.2229V9.62155H5.77849V18.2702Z"
        fill="#121E28"
      />
    </SvgIcon>
  );
}
