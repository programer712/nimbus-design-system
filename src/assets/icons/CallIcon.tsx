import { SvgIcon, SvgIconProps } from '@material-ui/core';
import React from 'react';

export function CallIcon({ fill = '#0064CC', ...props }: SvgIconProps) {
  return (
    <SvgIcon width={22} height={23} viewBox="0 0 22 23" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
      <path
        d="M16.921 22.5C7.712 22.488.25 15.025.238 5.816A4.983 4.983 0 015.221.833c.28-.002.56.023.835.076.265.039.527.104.78.195.362.127.63.436.704.812l1.484 6.5c.081.359-.025.734-.282.997-.14.152-.151.162-1.484.856a10.735 10.735 0 005.276 5.297c.704-1.343.715-1.354.866-1.495.264-.257.639-.363.997-.281l6.5 1.484c.364.084.66.35.78.704a4.7 4.7 0 01.271 1.614 4.983 4.983 0 01-5.027 4.908z"
        fill={fill}
      />
    </SvgIcon>
  );
}
