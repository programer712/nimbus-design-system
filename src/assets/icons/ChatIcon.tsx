import { SvgIcon, SvgIconProps } from '@material-ui/core';
import React from 'react';

export function ChatIcon({ fill = '#0064CC', ...props }: SvgIconProps) {
  return (
    <SvgIcon width="28" height="29" viewBox="0 0 28 29" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
      <path
        fillRule="evenodd"
        clipRule="evenodd"
        d="M20.875 3.6665V13.2915C20.875 14.804 19.6375 16.0415 18.125 16.0415H5.75L0.25 21.5415V3.6665C0.25 2.154 1.4875 0.916504 3 0.916504H18.125C19.6375 0.916504 20.875 2.154 20.875 3.6665ZM23.6248 6.41663H24.9998C26.5123 6.41663 27.7498 7.65413 27.7498 9.16663V28.4166L22.2498 22.9166H8.49976C6.98726 22.9166 5.74976 21.6791 5.74976 20.1666V18.7916H22.2498C23.006 18.7916 23.6248 18.1729 23.6248 17.4166V6.41663Z"
        fill={fill}
      />
    </SvgIcon>
  );
}
