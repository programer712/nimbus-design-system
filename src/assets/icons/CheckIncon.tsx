import { SvgIcon, SvgIconProps } from '@material-ui/core';
import React from 'react';

export function CheckIcon(props?: SvgIconProps) {
  return (
    <SvgIcon {...props}>
      <rect opacity="0.01" x="0.274065" y="0.27417" width="24" height="24" fill="#44AF69" />
      <path
        fillRule="evenodd"
        clipRule="evenodd"
        d="M2.34258 12.6851C2.34258 6.97307 6.97312 2.34253 12.6852 2.34253C15.4282 2.34253 18.0589 3.43219 19.9985 5.3718C21.9381 7.31142 23.0278 9.9421 23.0278 12.6851C23.0278 18.3972 18.3972 23.0277 12.6852 23.0277C6.97312 23.0277 2.34258 18.3972 2.34258 12.6851ZM12.4059 16.4188L17.1325 10.2132V10.1822C17.3579 9.88661 17.4085 9.49321 17.2652 9.15021C17.1219 8.8072 16.8065 8.56671 16.4378 8.51931C16.0691 8.47191 15.7031 8.62481 15.4777 8.92042L11.5682 14.0917L9.88233 11.9405C9.65509 11.6485 9.28937 11.5 8.92295 11.5509C8.55652 11.6017 8.24506 11.8442 8.10588 12.187C7.9667 12.5297 8.02096 12.9207 8.2482 13.2126L10.7718 16.4291C10.9692 16.6789 11.2705 16.8239 11.5889 16.8222C11.909 16.8214 12.2107 16.6724 12.4059 16.4188Z"
        fill="#44AF69"
      />
    </SvgIcon>
  );
}
