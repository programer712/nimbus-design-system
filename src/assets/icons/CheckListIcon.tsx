import { SvgIcon, SvgIconProps } from '@material-ui/core';
import React from 'react';

export function CheckListIcon(props?: SvgIconProps) {
  return (
    <svg width={24} height={24} viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
      <path
        d="M18.333 4H5.667A1.667 1.667 0 004 5.667v12.666A1.666 1.666 0 005.667 20h12.666A1.666 1.666 0 0020 18.333V5.667A1.666 1.666 0 0018.333 4zm-2 6.333h-3a.667.667 0 010-1.333h3a.667.667 0 110 1.333zM17 15.667a.667.667 0 01-.667.666h-3a.667.667 0 010-1.333h3a.666.666 0 01.667.667zm-5.467-7.934l-2 2.667a.667.667 0 01-.486.267H9a.667.667 0 01-.471-.196l-1-1a.667.667 0 11.942-.943l.457.457 1.539-2.052a.667.667 0 111.066.8zm0 6.667l-2 2.667a.68.68 0 01-1.004.071l-1-1a.667.667 0 01.942-.943l.457.457 1.539-2.052a.667.667 0 011.066.8z"
        fill="#0064CC"
      />
    </svg>
  );
}
