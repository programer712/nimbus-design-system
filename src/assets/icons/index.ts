export { ArrowIcon } from './ArrowIcon';
export { CalendarCallIcon } from './CalendarCallIcon';
export { CallIcon } from './CallIcon';
export { ChatIcon } from './ChatIcon';
export { CheckIcon } from './CheckIncon';
export { ChevronIcon } from './ChevronIcon';
export { CloseIcon } from './CloseIcon';
export { CogIcon } from './CogIcon';
export { CopyToClipboardIcon } from './CopyToClipboardIcon';
export { DangerIcon } from './DangerIcon';
export { DownloadIcon } from './DownloadIcon';
export { FileIcon } from './FileIcon';
export { GroupChatIcon } from './GroupChatIcon';
export { HomeIcon } from './HomeIcon';
export { IncomingCallIcon } from './IncomingCallIcon';
export { LocationIcon } from './LocationIcon';
export { LogoutIcon } from './LogoutIcon';
export { MessageIcon } from './MessageIcon';
export { MissedCallIcon } from './MissedCallIcon';
export { OutgoingCallIcon } from './OutgoingCallIcon';
export { PlayVideoIcon } from './PlayVideoIcon';
export { QuestionMarkCallIcon } from './QuestionMarkCallIcon';
export { SearchIcon } from './SearchIcon';
export { SortOrderIcon } from './SortOrderIcon';
export { TimerCallIcon } from './TimerCallIcon';
export { UnknownAppIcon } from './UnknownAppIcon';
export { VideoCameraCallIcon } from './VideoCameraCallIcon';
export { InfoIcon } from './InfoIcon';
export { JumbotronIcon } from './JumbotronIcon';
export { JumbotronSkeletonIcon } from './JumbotronSkeletonIcon';
export { MobileIcon } from './MobileIcon';
export { CheckListIcon } from './CheckListIcon';
export { DatabaseIcon } from './DatabaseIcon';
export { MagnifyingGlassIcon } from './MagnifyingGlassIcon';
export { WarningTriangleIcon } from './WarningTriangleIcon';
export { NoResultsIcon } from './NoResultsIcon';
export { NoDeviceInfoIcon } from './NoDeviceInfoIcon';
