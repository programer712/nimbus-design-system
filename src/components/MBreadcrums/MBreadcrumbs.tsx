import { last } from 'lodash';
import clsx from 'clsx';
import React from 'react';
import PropTypes from 'prop-types';
import { Link as RouterLink } from 'react-router-dom';
import { makeStyles } from '@material-ui/core/styles';
import { Box, Link, Breadcrumbs } from '@material-ui/core';

// ----------------------------------------------------------------------

const useStyles = makeStyles((theme) => ({
  root: {
    '& $separator': {
      width: 4,
      height: 4,
      borderRadius: '50%',
      marginLeft: theme.spacing(2),
      marginRight: theme.spacing(2),
      backgroundColor: theme.palette.text.disabled,
    },
  },
  link: {
    lineHeight: 2,
    display: 'flex',
    alignItems: 'center',
    color: theme.palette.grey[500],
    '& > div': { display: 'inherit' },
    '& svg': { width: 20, height: 20 },
  },
  separator: {},
}));

// ----------------------------------------------------------------------

export interface LinkProp {
  href: string;
  name: string;
  icon?: React.ReactElement;
}

function LinkItem({ link }: { link: LinkProp }) {
  const classes = useStyles();
  const { href, name, icon } = link;
  return (
    <Link to={href} key={name} variant="body2" color="textPrimary" component={RouterLink} className={classes.link}>
      {icon && <Box sx={{ mr: 1 }}> {icon} </Box>}
      {name}
    </Link>
  );
}

MBreadcrumbs.propTypes = {
  links: PropTypes.array.isRequired,
  activeLast: PropTypes.bool,
  className: PropTypes.string,
};

interface MBreadCrumbsProp {
  links: LinkProp[];
  activeLast?: boolean;
  className?: string;
}

function MBreadcrumbs({ links, activeLast = false, className, ...other }: MBreadCrumbsProp) {
  const classes = useStyles();
  const currentLink = last(links)?.name;

  const listDefault = links.map((link, index) => <LinkItem key={index} link={link} />);
  const listActiveLast = links.map((link, index) => (
    <div key={link.name}>
      {link.name !== currentLink ? (
        <LinkItem key={index} link={link} />
      ) : (
        <Box
          component="p"
          sx={{
            maxWidth: 260,
            overflow: 'hidden',
            typography: 'body2',
            whiteSpace: 'nowrap',
            color: 'text.disabled',
            textOverflow: 'ellipsis',
          }}
        >
          {currentLink}
        </Box>
      )}
    </div>
  ));

  return (
    <Breadcrumbs
      separator={<span />}
      classes={{ separator: classes.separator }}
      className={clsx(classes.root, className)}
      {...other}
    >
      {activeLast ? listDefault : listActiveLast}
    </Breadcrumbs>
  );
}

export default MBreadcrumbs;
