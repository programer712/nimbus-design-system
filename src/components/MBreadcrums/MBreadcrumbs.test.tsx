import { cleanup, render } from '@testing-library/react';
import { Breadcrumbs } from './MBreadcrumbs.stories';
import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { ArrowIcon, CogIcon } from '../../assets/icons';

afterEach(cleanup);

it('should render without errors when provided the minimum required props', () => {
  const { asFragment, container } = render(
    <BrowserRouter>
      <Breadcrumbs
        key={'testId'}
        links={[
          { name: 'Step 1', href: '', icon: <CogIcon /> },
          { name: 'Step 2', href: '', icon: <ArrowIcon /> },
        ]}
        className="random-string-name"
      />
    </BrowserRouter>
  );

  expect(container).toBeInTheDocument();
  expect(asFragment()).toMatchSnapshot();
});

it('should render without errors when activeLast is true', () => {
  const { asFragment, container } = render(
    <BrowserRouter>
      <Breadcrumbs
        activeLast
        links={[
          { name: 'Step 1', href: '', icon: <CogIcon /> },
          { name: 'Step 2', href: '', icon: <ArrowIcon /> },
        ]}
        className="random-string-name"
      />
    </BrowserRouter>
  );

  expect(container).toBeInTheDocument();
  expect(asFragment()).toMatchSnapshot();
});
