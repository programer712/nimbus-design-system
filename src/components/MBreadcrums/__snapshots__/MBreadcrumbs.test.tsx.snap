// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without errors when activeLast is true 1`] = `
<DocumentFragment>
  <nav
    class="MuiTypography-root MuiTypography-body1 MuiBreadcrumbs-root makeStyles-root-4 random-string-name css-63vwl1-MuiTypography-root-MuiBreadcrumbs-root"
  >
    <ol
      class="MuiBreadcrumbs-ol css-4pdmu4-MuiBreadcrumbs-ol"
    >
      <li
        class="MuiBreadcrumbs-li"
      >
        <a
          class="MuiTypography-root MuiTypography-body2 MuiLink-root MuiLink-underlineHover makeStyles-link-5 css-1mlnpg5-MuiTypography-root-MuiLink-root"
          href="/"
        >
          <div
            class="css-wu59qs"
          >
             
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <rect
                fill="#BDC3C8"
                height="24"
                opacity="0.01"
                transform="rotate(180 24 24)"
                width="24"
                x="24"
                y="24"
              />
              <circle
                cx="12"
                cy="12"
                fill="#BDC3C8"
                r="1.05"
              />
              <path
                clip-rule="evenodd"
                d="M17.061 10.159H17.824C18.4746 10.1628 19 10.6914 19 11.342V12.7C18.9966 13.3307 18.4963 13.8465 17.866 13.869H17.103C17.0112 13.8767 16.9322 13.9367 16.9 14.023C16.8489 14.1018 16.8489 14.2032 16.9 14.282L17.411 14.793C17.632 15.0125 17.7563 15.311 17.7563 15.6225C17.7563 15.934 17.632 16.2325 17.411 16.452L16.473 17.39C16.2561 17.6132 15.9582 17.7394 15.647 17.74C15.3344 17.7359 15.0357 17.6104 14.814 17.39L14.282 16.851C14.2032 16.7999 14.1018 16.7999 14.023 16.851C13.918 16.893 13.841 16.963 13.841 17.061V17.824C13.8371 18.4746 13.3086 19 12.658 19H11.335C10.7016 19.0004 10.1816 18.499 10.159 17.866V17.103C10.1513 17.0112 10.0913 16.9322 10.005 16.9C9.91754 16.844 9.80546 16.844 9.718 16.9L9.179 17.418C8.95954 17.639 8.66096 17.7633 8.3495 17.7633C8.03804 17.7633 7.73946 17.639 7.52 17.418L6.582 16.466C6.35774 16.2469 6.23152 15.9465 6.232 15.633C6.2361 15.3204 6.36162 15.0217 6.582 14.8L7.149 14.282C7.20009 14.2032 7.20009 14.1018 7.149 14.023C7.107 13.918 7.037 13.841 6.939 13.841H6.176C5.52538 13.8371 4.99999 13.3086 5 12.658V11.335C5 10.6855 5.52651 10.159 6.176 10.159H6.897C6.98876 10.1513 7.06782 10.0913 7.1 10.005C7.15604 9.91754 7.15604 9.80546 7.1 9.718L6.582 9.2C6.35446 8.97977 6.226 8.67666 6.226 8.36C6.226 8.04334 6.35446 7.74023 6.582 7.52L7.541 6.582C7.75793 6.3588 8.05575 6.23261 8.367 6.232C8.67957 6.2361 8.9783 6.36162 9.2 6.582L9.718 7.149C9.79678 7.20009 9.89822 7.20009 9.977 7.149C10.082 7.107 10.159 7.037 10.159 6.939V6.176C10.1628 5.52538 10.6914 4.99999 11.342 5H12.7C13.3358 5.01893 13.8413 5.53994 13.841 6.176V6.897C13.8487 6.98876 13.9087 7.06782 13.995 7.1C14.0825 7.15604 14.1945 7.15604 14.282 7.1L14.821 6.582C15.0405 6.361 15.339 6.23671 15.6505 6.23671C15.962 6.23671 16.2605 6.361 16.48 6.582L17.418 7.541C17.6416 7.76054 17.7677 8.06065 17.768 8.374C17.769 8.68557 17.6426 8.984 17.418 9.2L16.851 9.718C16.7999 9.79678 16.7999 9.89822 16.851 9.977C16.893 10.082 16.963 10.159 17.061 10.159ZM9.7365 12.9376C10.1157 13.8531 11.0091 14.45 12 14.45C13.3531 14.45 14.45 13.3531 14.45 12C14.45 11.0091 13.8531 10.1157 12.9376 9.7365C12.0221 9.35728 10.9683 9.5669 10.2676 10.2676C9.5669 10.9683 9.35728 12.0221 9.7365 12.9376Z"
                fill-rule="evenodd"
              />
               
            </svg>
             
          </div>
          Step 1
        </a>
      </li>
      <li
        aria-hidden="true"
        class="makeStyles-separator-6 MuiBreadcrumbs-separator css-1wuw8dw-MuiBreadcrumbs-separator"
      >
        <span />
      </li>
      <li
        class="MuiBreadcrumbs-li"
      >
        <a
          class="MuiTypography-root MuiTypography-body2 MuiLink-root MuiLink-underlineHover makeStyles-link-5 css-1mlnpg5-MuiTypography-root-MuiLink-root"
          href="/"
        >
          <div
            class="css-wu59qs"
          >
             
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 20L13.41 18.59L7.83 13L20 13L20 11L7.83 11L13.42 5.42L12 4L4 12L12 20Z"
              />
            </svg>
             
          </div>
          Step 2
        </a>
      </li>
    </ol>
  </nav>
</DocumentFragment>
`;

exports[`should render without errors when provided the minimum required props 1`] = `
<DocumentFragment>
  <nav
    class="MuiTypography-root MuiTypography-body1 MuiBreadcrumbs-root makeStyles-root-1 random-string-name css-63vwl1-MuiTypography-root-MuiBreadcrumbs-root"
  >
    <ol
      class="MuiBreadcrumbs-ol css-4pdmu4-MuiBreadcrumbs-ol"
    >
      <li
        class="MuiBreadcrumbs-li"
      >
        <div>
          <a
            class="MuiTypography-root MuiTypography-body2 MuiLink-root MuiLink-underlineHover makeStyles-link-2 css-1mlnpg5-MuiTypography-root-MuiLink-root"
            href="/"
          >
            <div
              class="css-wu59qs"
            >
               
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <rect
                  fill="#BDC3C8"
                  height="24"
                  opacity="0.01"
                  transform="rotate(180 24 24)"
                  width="24"
                  x="24"
                  y="24"
                />
                <circle
                  cx="12"
                  cy="12"
                  fill="#BDC3C8"
                  r="1.05"
                />
                <path
                  clip-rule="evenodd"
                  d="M17.061 10.159H17.824C18.4746 10.1628 19 10.6914 19 11.342V12.7C18.9966 13.3307 18.4963 13.8465 17.866 13.869H17.103C17.0112 13.8767 16.9322 13.9367 16.9 14.023C16.8489 14.1018 16.8489 14.2032 16.9 14.282L17.411 14.793C17.632 15.0125 17.7563 15.311 17.7563 15.6225C17.7563 15.934 17.632 16.2325 17.411 16.452L16.473 17.39C16.2561 17.6132 15.9582 17.7394 15.647 17.74C15.3344 17.7359 15.0357 17.6104 14.814 17.39L14.282 16.851C14.2032 16.7999 14.1018 16.7999 14.023 16.851C13.918 16.893 13.841 16.963 13.841 17.061V17.824C13.8371 18.4746 13.3086 19 12.658 19H11.335C10.7016 19.0004 10.1816 18.499 10.159 17.866V17.103C10.1513 17.0112 10.0913 16.9322 10.005 16.9C9.91754 16.844 9.80546 16.844 9.718 16.9L9.179 17.418C8.95954 17.639 8.66096 17.7633 8.3495 17.7633C8.03804 17.7633 7.73946 17.639 7.52 17.418L6.582 16.466C6.35774 16.2469 6.23152 15.9465 6.232 15.633C6.2361 15.3204 6.36162 15.0217 6.582 14.8L7.149 14.282C7.20009 14.2032 7.20009 14.1018 7.149 14.023C7.107 13.918 7.037 13.841 6.939 13.841H6.176C5.52538 13.8371 4.99999 13.3086 5 12.658V11.335C5 10.6855 5.52651 10.159 6.176 10.159H6.897C6.98876 10.1513 7.06782 10.0913 7.1 10.005C7.15604 9.91754 7.15604 9.80546 7.1 9.718L6.582 9.2C6.35446 8.97977 6.226 8.67666 6.226 8.36C6.226 8.04334 6.35446 7.74023 6.582 7.52L7.541 6.582C7.75793 6.3588 8.05575 6.23261 8.367 6.232C8.67957 6.2361 8.9783 6.36162 9.2 6.582L9.718 7.149C9.79678 7.20009 9.89822 7.20009 9.977 7.149C10.082 7.107 10.159 7.037 10.159 6.939V6.176C10.1628 5.52538 10.6914 4.99999 11.342 5H12.7C13.3358 5.01893 13.8413 5.53994 13.841 6.176V6.897C13.8487 6.98876 13.9087 7.06782 13.995 7.1C14.0825 7.15604 14.1945 7.15604 14.282 7.1L14.821 6.582C15.0405 6.361 15.339 6.23671 15.6505 6.23671C15.962 6.23671 16.2605 6.361 16.48 6.582L17.418 7.541C17.6416 7.76054 17.7677 8.06065 17.768 8.374C17.769 8.68557 17.6426 8.984 17.418 9.2L16.851 9.718C16.7999 9.79678 16.7999 9.89822 16.851 9.977C16.893 10.082 16.963 10.159 17.061 10.159ZM9.7365 12.9376C10.1157 13.8531 11.0091 14.45 12 14.45C13.3531 14.45 14.45 13.3531 14.45 12C14.45 11.0091 13.8531 10.1157 12.9376 9.7365C12.0221 9.35728 10.9683 9.5669 10.2676 10.2676C9.5669 10.9683 9.35728 12.0221 9.7365 12.9376Z"
                  fill-rule="evenodd"
                />
                 
              </svg>
               
            </div>
            Step 1
          </a>
        </div>
      </li>
      <li
        aria-hidden="true"
        class="makeStyles-separator-3 MuiBreadcrumbs-separator css-1wuw8dw-MuiBreadcrumbs-separator"
      >
        <span />
      </li>
      <li
        class="MuiBreadcrumbs-li"
      >
        <div>
          <p
            class="css-13uwyqo"
          >
            Step 2
          </p>
        </div>
      </li>
    </ol>
  </nav>
</DocumentFragment>
`;
